pkg/controller/node/nodecontroller.go

```diff
@@ -573,6 +668,42 @@ func (nc *NodeController) monitorNodeStatus() error {
 	return nil
 }
 
+// evictIntolerablePods checks for given node if there's any pod that
```

Comment:
I'm probably missing something, but I don't see how we cancel eviction in a case when Node comes back.